libnavigation-ui/src/main/java/com/mapbox/navigation/ui/instruction/InstructionView.java

```diff
@@ -279,6 +280,7 @@ public void unsubscribe() {
       navigationViewModel.retrieveIsOffRoute().removeObservers(lifecycleOwner);
       navigationViewModel.retrieveRouteJunctionModelUpdates().removeObservers(lifecycleOwner);
     }
+    ImageCreator.getInstance().shutdown();
```

Comment:
@abhishek1508 this on purpose? looks like a legit fix, but surfacing cause it looks unrelated

Reply:
@kmadsen Yeah this was a part of package moving change.

Reply:
spoke offline, on purpose 👍